src/Locale/de/cake.po

```diff
@@ -246,7 +246,7 @@ msgstr "Start"
 
 #: View/Widget/DateTimeWidget.php:540
 msgid "January"
-msgstr "Januar"
+msgstr "Jänner"
```

Comment:
First of all this change is non BC. People don't assume `de` to be `de_AT` and the month names are actually being used a lot.

Well I assume de, when conflict arises maps to de_DE like en, when conflict arises maps to en_US (not say en_GB). Please @dereuromark @ravage84 take a look.

I am not sure if localized supports fallbacks e.g. so that you can create de.po and de_AT.po with just "Jänner" inside.